### PR TITLE
refresh IpFabric doc

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 1]
```diff
@@ -205,28 +205,41 @@ yum install centreon-plugin-Applications-Ipfabric-Api
...
+	--warning-paths-state-all='' \
+	--critical-paths-state-all='' \
+	--warning-paths-state-part='' \
+	--critical-paths-state-part='' \
+	--warning-paths-state-none='' \
+	--critical-paths-state-none='' \
+	--warning-paths-state-error='' \
+	--critical-paths-state-error='' \
+	--http-backend=curl\
+
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-ipfabric-api.md:236]:
```suggestion
/usr/lib/centreon/plugins/centreon_ipfabric_api.pl \
    --plugin=apps::ipfabric::plugin \
    --mode=path-verification \
    --api-key='****' \
    --hostname='demo1.eu.ipfabric.io' \
    --verbose
```

Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-ipfabric-api.md:236]: The previous example was better

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 2]
```diff
@@ -205,28 +205,41 @@ yum install centreon-plugin-Applications-Ipfabric-Api
...
-source 10.35.119.172:- destination 10.193.0.132:- [protocol: icmp] state: none [expected state: all]
-source 10.35.119.172/26:1024-65535 destination 10.193.0.132:1-60000 [protocol: tcp] state: none [expected state: none]
-source 10.35.119.172/24:1024-65535 destination 10.193.0.132:80 [protocol: tcp] state: none [expected state: none]
-source 10.72.126.112:1024-65535 destination 10.66.121.110:80,443 [protocol: tcp] state: none [expected state: all]
-source 10.66.126.112:10000 destination 10.38.115.190:443 [protocol: tcp] state: none [expected state: all]
-source 10.35.119.172/28:1024-65535 destination 10.193.0.132/28:1-60000 [protocol: tcp] state: none [expected state: none]
+OK:        | 'paths.detected.count'=88;;;0; 'paths.mismatch.count'=93;;;0; 'paths.state.all.count'=61;;;0; 'paths.state.part.count'=61;;;0; 'paths.state.none.count'=64;;;0; 'paths.state.error.count'=90;;;0;
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-ipfabric-api.md:242]:
```suggestion
CRITICAL: source 10.35.119.172:- destination 10.193.0.132:- [protocol: icmp] state: none [expected state: all] - source 10.72.126.112:1024-65535 destination 10.66.121.110:80,443 [protocol: tcp] state: none [expected state: all] - source 10.66.126.112:10000 destination 10.38.115.190:443 [protocol: tcp] state: none [expected state: all] | 'paths.detected.count'=6;;;0; 'paths.mismatch.count'=3;;;0; 'paths.state.all.count'=0;;;0; 'paths.state.part.count'=0;;;0; 'paths.state.none.count'=6;;;0; 'paths.state.error.count'=0;;;0;
source 10.35.119.172:- destination 10.193.0.132:- [protocol: icmp] state: none [expected state: all]
source 10.35.119.172/26:1024-65535 destination 10.193.0.132:1-60000 [protocol: tcp] state: none [expected state: none]
source 10.35.119.172/24:1024-65535 destination 10.193.0.132:80 [protocol: tcp] state: none [expected state: none]
source 10.72.126.112:1024-65535 destination 10.66.121.110:80,443 [protocol: tcp] state: none [expected state: all]
source 10.66.126.112:10000 destination 10.38.115.190:443 [protocol: tcp] state: none [expected state: all]
source 10.35.119.172/28:1024-65535 destination 10.193.0.132/28:1-60000 [protocol: tcp] state: none [expected state: none]
```

File: pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 3]
```diff
@@ -174,58 +174,71 @@ yum install centreon-plugin-Applications-Ipfabric-Api
...
+	--warning-paths-state-all='' \
+	--critical-paths-state-all='' \
+	--warning-paths-state-part='' \
+	--critical-paths-state-part='' \
+	--warning-paths-state-none='' \
+	--critical-paths-state-none='' \
+	--warning-paths-state-error='' \
+	--critical-paths-state-error='' \
+	--http-backend=curl\
+
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md:235]:
```suggestion
    --plugin=apps::ipfabric::plugin \
    --mode=path-verification \
    --api-key='****' \
    --hostname='demo1.eu.ipfabric.io' \
    --verbose
```

File: pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md [hunk 4]
```diff
@@ -174,58 +174,71 @@ yum install centreon-plugin-Applications-Ipfabric-Api
...
-source 10.35.119.172:- destination 10.193.0.132:- [protocol: icmp] state: none [expected state: all]
-source 10.35.119.172/26:1024-65535 destination 10.193.0.132:1-60000 [protocol: tcp] state: none [expected state: none]
-source 10.35.119.172/24:1024-65535 destination 10.193.0.132:80 [protocol: tcp] state: none [expected state: none]
-source 10.72.126.112:1024-65535 destination 10.66.121.110:80,443 [protocol: tcp] state: none [expected state: all]
-source 10.66.126.112:10000 destination 10.38.115.190:443 [protocol: tcp] state: none [expected state: all]
-source 10.35.119.172/28:1024-65535 destination 10.193.0.132/28:1-60000 [protocol: tcp] state: none [expected state: none]
+OK:        | 'paths.detected.count'=88;;;0; 'paths.mismatch.count'=93;;;0; 'paths.state.all.count'=61;;;0; 'paths.state.part.count'=61;;;0; 'paths.state.none.count'=64;;;0; 'paths.state.error.count'=90;;;0;
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-ipfabric-api.md:241]:
```suggestion
CRITICAL: source 10.35.119.172:- destination 10.193.0.132:- [protocol: icmp] state: none [expected state: all] - source 10.72.126.112:1024-65535 destination 10.66.121.110:80,443 [protocol: tcp] state: none [expected state: all] - source 10.66.126.112:10000 destination 10.38.115.190:443 [protocol: tcp] state: none [expected state: all] | 'paths.detected.count'=6;;;0; 'paths.mismatch.count'=3;;;0; 'paths.state.all.count'=0;;;0; 'paths.state.part.count'=0;;;0; 'paths.state.none.count'=6;;;0; 'paths.state.error.count'=0;;;0;
source 10.35.119.172:- destination 10.193.0.132:- [protocol: icmp] state: none [expected state: all]
source 10.35.119.172/26:1024-65535 destination 10.193.0.132:1-60000 [protocol: tcp] state: none [expected state: none]
source 10.35.119.172/24:1024-65535 destination 10.193.0.132:80 [protocol: tcp] state: none [expected state: none]
source 10.72.126.112:1024-65535 destination 10.66.121.110:80,443 [protocol: tcp] state: none [expected state: all]
source 10.66.126.112:10000 destination 10.38.115.190:443 [protocol: tcp] state: none [expected state: all]
source 10.35.119.172/28:1024-65535 destination 10.193.0.132/28:1-60000 [protocol: tcp] state: none [expected state: none]
```

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**